mod/settings.php

```diff
@@ -149,9 +149,7 @@ function settings_post(App $a)
 		check_form_security_token_redirectOnErr('/settings/oauth', 'settings_oauth');
 
 		$key = $_POST['remove'];
-		q("DELETE FROM tokens WHERE id='%s' AND uid=%d",
-			dbesc($key),
-			local_user());
+		dba::delete('tokens', ['id' => $key]);
```

Comment:
There's no previous ownership test, I believe the `uid` condition should be kept in place.